MediaGalleryUi/view/adminhtml/web/css/source/_module.less

```diff
@@ -368,3 +368,21 @@
         margin-right: 1.4rem;
     }
 }
+
+.media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__m) {
+    .media-gallery-image-details-modal {
+        .image-details {
+            .lib-vendor-prefix-display();
```

Comment:
I believe we can remove this and set it on `.image-details` outside the media query, what do you think?